prow/cmd/deck/static/pr.html

```diff
@@ -41,6 +41,27 @@
     <main id="main-container" class="mdl-layout__content">
     </main>
 </div>
+<dialog id="search-dialog" class="pr-status-dialog mdl-dialog">
+    <h4 class="mdl-dialog__title">Github search query</h4>
+    <div class="mdl-dialog__content">
+        <p>You can use the Github search query to search for <strong>pull requests</strong> using Github search syntax.</p>
+        <p>Notice:</p>
+        <ol>
+            <li>
+                <p>The search feature is used to search <strong>pull requests</strong>. We enforce this by dropping any <code>is:issue</code> or <code>type:issue</code> token in the query and will append <code>is:pr</code> to the query if <code>is:pr</code> does not exist.</p>
+            </li>
+            <li>
+                <p>You should always specify organizations or repos of the pull requests. If you do not mention any organziation or repository in the query, we will search for pull requests that are configured with either <strong>Prow</strong> or <strong>Tide</strong>.</p>
```

Comment:
We can probably drop the first sentence.
Also,
s/pull requests that are configured with/pull requests in repositories that use/